collection refreshes on book add and remove

LWcaveman opened the pull request.

LWcaveman committed:
collection refreshes on book add and remove

Thunderpig851 reviewed (approved):
Looks good. Go ahead and resolve locally if possible.

LWcaveman committed:
resolved book.js and app.js conflicts
Merge branch 'main' of https://github.com/Blue-Ocean-Koi-Fish/lit-hub into collectionsUpdate
